schedule/yast/yast_hostname/yast_hostname+dhcp_hostname_x86_64.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: yast_hostname+dhcp_hostname
+description: >
+  Test suite uses default option to set dhcp (using ifgf=*=dhcp), hostname.
+  Test validates if installation can successfully start in case of usage of
+  these parameters.
+vars:
+  EXIT_AFTER_START_INSTALL: '1'
+  EXPECTED_INSTALL_HOSTNAME: dhcphostname
+  EXTRABOOTPARAMS: ifcfg=*=dhcp
+  NICTYPE_USER_OPTIONS: hostname=dhcphostname
+  YUI_REST_API: 1
+schedule:
+  local_user: []
+  booting: []
+  default_systemd_target: []
+  installation_settings:
+    - installation/hostname_inst
```

Comment:
let's hook this with timezone and put it the first, so the below we will have only empty hooks for which we can create some feature later to avoid writing them. Please apply the same in the other *hostname* test suites.

Reply:
changed.